build_configs/macos/build.spec

```diff
@@ -1,14 +1,19 @@
 # -*- mode: python ; coding: utf-8 -*-
+from PyInstaller.utils.hooks import copy_metadata
+
+datas = [
+    ('../../staking_deposit/key_handling/key_derivation/word_lists/*.txt', './staking_deposit/key_handling/key_derivation/word_lists/'),
+    ('../../staking_deposit/intl', './staking_deposit/intl'),
+]
+datas += copy_metadata('py_ecc')
```

Comment:
Curious why we are doing this as well

Reply:
Both py_ecc and ssz are using importlib for metadata reflection/inspection during module load. It might have been added in a recent version. In any case, the app triggers an unhandled exception if that data is not included in the bundled binary.